src/app/zkapps_examples/zkapps_examples.ml

```diff
@@ -128,13 +155,37 @@ module Account_update_under_construction = struct
                 ; Some a5
                 ; Some a6
                 ; Some a7
+                ; Some a8
```

Comment:
Could you refactor this using List.map to wrap everything in a `Some`?

Reply:
It turns out it can even just be this:
```ocaml
      let set_full_state app_state (_t : t) =
            { app_state =
                Pickles_types.Vector.map ~f:(fun a -> Some a)
                @@ Pickles_types.Vector.of_list_and_length_exn app_state Zkapp_state.Max_state_size.n
            }

```
no ugly literals and it shouldn't even break if we increase the app_state again later!